go-controller/pkg/metrics/master.go

```diff
@@ -698,6 +728,520 @@ func getPodUIDFromPortBinding(row *sbdb.PortBinding) kapimtypes.UID {
 	return kapimtypes.UID(podUID)
 }
 
+const (
+	updateOVNMeasurementChSize = 500
+	deleteOVNMeasurementChSize = 50
+	processChSize              = 1000
+	nbGlobalTable              = "NB_Global"
+	// we currently do not handle overflow, so we stop serving when close to overflow
```

Comment:
LOL yeah @dceara told me about this. We need to get that fixed in OVN. Maybe add a FIXME here.

Reply:
done